apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -76,10 +76,14 @@ function Navbar({ identity }: NavbarProps) {
 					<NavLinkItem href="/">Schedule</NavLinkItem>
 					<NavLinkItem href="/">Resources</NavLinkItem>
 					<NavLinkItem href="/">Stage</NavLinkItem> */}
-					{!status && <NavLinkItem href="/apply">Apply</NavLinkItem>}
-					{status !== null && (
-						<NavLinkItem href="/portal">Portal</NavLinkItem>
-					)}
+					{!status &&
+						(deadlinePassed ? (
+							<Button text="Applications closed" disabled />
+						) : (
+							<NavLinkItem href="/apply">Apply</NavLinkItem>
+						))}
```

Comment:
Thought: not sure if having a button in the middle of the Navigation is the best design. Maybe we could just remove the Apply link entirely? Might want to consult graphics as well.